src/routes/paths.ts

```diff
@@ -27,3 +27,25 @@ export const COMPONENT_ACTIVITY_CHILD_TAB_PATH = COMPONENT_ACTIVITY_PATH.extend(
 export const COMMIT_LIST_PATH = APPLICATION_DETAILS_PATH.extend('commit');
 
 export const COMMIT_DETAILS_PATH = COMMIT_LIST_PATH.extend(`:${RouterParams.commitName}`);
+
+// Integration test paths
+
+export const INTEGRATION_TEST_LIST_PATH = APPLICATION_DETAILS_PATH.extend('integrationtests');
```

Comment:
move routes for integration to the `page-routes`.

Reply:
moved